### ISSUE TITLE
Cannot extend native interfaces

### ISSUE THREAD

**Issue by darzu** [microsoft/pxt#6859]
This interface:
```
    export interface MinimapImage extends Image {
        minimapScale: MinimapScale;
    }
```
produces this runtime error:
![image](https://user-images.githubusercontent.com/6453828/79400071-f942f600-7f39-11ea-861a-7e1b92ea6d07.png)

in this program:
https://makecode.com/_gxCUs9bgWV5R

Using this extension:
https://github.com/darzu/arcade-minimap

**Comment by pelikhan** [comment]
Image is a “native” type, implemented in C++ and JS sim. I am not sure it is a good idea to try to extend it.

________________________________

**Comment by darzu** [comment]
Yeah, Michal and I chatted a bit and it's probably best to just show an error for now.
